source/apiVolontaria/volunteer/serializers.py

```diff
@@ -433,7 +433,6 @@ class ParticipationAdminSerializer(serializers.ModelSerializer):
     standby = serializers.BooleanField()
     user = UserBasicSerializer(
         read_only=True,
```

Comment:
Actually the field is defined as `read_only` but we can write it since you overwrite the view.